src/utils/components/DiskModal/DiskModal.tsx

```diff
@@ -193,6 +188,13 @@ const DiskModal: FC<DiskModalProps> = ({
       obj={updatedVirtualMachine}
       onSubmit={handleSubmit}
     >
+      {isVMRunning && (
+        <PendingChangesAlert title={t(' Adding hot plugged disk')}>
+          {t(
+            'Additional disks types and interfaces are available when the VirtualMachine is stopped.',
```

Comment:
I think this would sound better:
`Additional disk types and interfaces become available when the VirtualMachine is stopped.`

In the original sentence, the word "disks" should be "disk" to maintain grammatical agreement with "types and interfaces". 
Also, the verb "are" should be changed to "become" to convey a smoother flow of action.
@pcbailey would you agree on this?

Reply:
Well, the sentence used here was recommended to me by Yifat and Ronen in the jira ticket discussion. We also pinged Avital to make sure about it, but she hasn't responded.

Reply:
I'll go with what you did for now, and if needed to be changed, let the docs team open a bug for it